README.md

```diff
@@ -92,6 +92,58 @@ Lily said,
 achieved tok/s:  359.66149506346966
 ```
 
+## play with Tinyllama-1.1B-Chat-v0.2
+First, navigate to the folder when you keep your projects and clone this repository to this folder:
+```bash
+git clone https://github.com/tairov/llama2.mojo.gits
+```
+
+Then, open the repository folder:
+
+```bash
+cd llama2.mojo
+```
+
+Now, let's download the model and the tokenizer
+
+```bash
+wget https://huggingface.co/kirp/TinyLlama-1.1B-Chat-v0.2-bin/resolve/main/tok_tl-chat.bin
```

Comment:
I will do this later because my convertor is different from the default convertor in llama2.c. I will make a pull request to llama2.c later.

Reply:
The reason why I support this model because it's the smallest **Billion parameter llama architecture** model. Since our llama2.mojo is run on fp32, we need at least 4G ram to run the model. So, at this stage we can not support other model like Openllama3B, llama2chat7B, Vicuna13B, etc.